application/common/middlewares.py

```diff
@@ -11,9 +15,17 @@ def unhandled_exception_logging(handler, event, context: LambdaContext):
     try:
         response = handler(event, context)
         return response
+    except ValidationException as err:
+        logger.exception("Validation Error", extra={"error": err, "event": event})
+        return
+    except ClientError as err:
+        error_code = err.response["Error"]["Code"]
+        error_msg = err.response["Error"]["Message"]
+        logger.exception(f"Boto3 Client Error - '{error_code}': {error_msg}", extra={"error": err, "event": event})
+        raise err
     except BaseException as err:
-        logger.exception("Something went wrong", extra={"error": err, "event": event})
-        raise
+        logger.exception("Something went wrong.", extra={"error": err, "event": event})
```

Comment:
can we change this to ...
`logger.exception(f"Something went wrong - {err.message}", extra={"error": err, "event": event})